browser.js

```diff
@@ -193,9 +193,7 @@ define(function(require, exports, module) {
                 
                 var src = getIframeSrc(iframe);
                 var path = calcRootedPath(cleanIframeSrc(src));
-                
-                tab.title = 
-                tab.tooltip = "[B] " + path;
```

Comment:
i think it was needed for the case when user clicks on a link inside previewed page, try creating 
1.html and 2.html with links to each other and use live preview on standalone.